airbyte-integrations/connectors/source-facebook-marketing/pyproject.toml

```diff
@@ -18,7 +18,7 @@ include = "source_facebook_marketing"
 [tool.poetry.dependencies]
 python = "^3.10,<3.12"
 airbyte-cdk = "^5"
-facebook-business = "19.0.0"
+facebook-business = "21.0.5"
```

Comment:
There is a `v22` api version already: https://developers.facebook.com/docs/graph-api/changelog/version22.0?locale=en_EN I wonder when the `v21` would be deprecated and how much time do we have ?

Reply:
We have started testing on v21 and the priority is to be ready for the API deprecation on Feb 4th with is very soon. I would focus on closing with v21 and we can add capabilities for v22 later. WDYT?

Reply:
I'm fine with this. Thanks @maxi297